.vscode/settings.json

```diff
@@ -21,5 +21,16 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "cSpell.words": [
```

Comment:
🤔

Reply:
yeah. didnt know how I flt about it. Will switch to User settings I think.